install/install.sh

```diff
@@ -114,8 +114,22 @@ function check_deb_packages {
           echo "checking $i"
           if [ $(dpkg-query -W -f='${Status}' $i | grep -c "ok installed") -eq 0 ]
           then
-            missing+=( $i )
-            halt=$((halt+1))
+            echo ""
```

Comment:
I'm having some doubts about installing debian packages from a script.

For a total amateur, it's good, but I think I would be annoyed if a script took this liberty on my system, especially if a conflicting dependency broke something or clobbered another working sub-system. 

I think I'd rather prompt users to do it with the resume message rather than taking this "liberty."

Reply:
Ok. I'll just skip the install attempt and display the "manual install..." plus call the abort method. I'll update the PR with those changes shortly